wfe/web-front-end.go

```diff
@@ -712,25 +716,24 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(response http.ResponseWriter, requ
 
 	certStatus, err := wfe.SA.GetCertificateStatus(serial)
 	if err != nil {
-		logEvent.Error = err.Error()
+		logEvent.AddError("unable to get certificate status: %s", err)
 		wfe.sendError(response, "Certificate status not yet available", err, http.StatusNotFound)
 		return
 	}
 	logEvent.Extra["CertificateStatus"] = certStatus.Status
 
 	if certStatus.Status == core.OCSPStatusRevoked {
-		logEvent.Error = "Certificate already revoked"
-		wfe.sendError(response, logEvent.Error, "", http.StatusConflict)
+		logEvent.AddError("Certificate already revoked: %#v", serial)
+		wfe.sendError(response, "Certificate already revoked", "", http.StatusConflict)
 		return
 	}
 
 	// TODO: Implement method of revocation by authorizations on account.
 	if !(core.KeyDigestEquals(requestKey, parsedCertificate.PublicKey) ||
 		registration.ID == cert.RegistrationID) {
-		logEvent.Error = "Revocation request must be signed by private key of cert to be revoked, or by the account key of the account that issued it."
-		wfe.log.Debug("Key mismatch for revoke")
+		logEvent.AddError("Revocation request must be signed by private key of cert to be revoked, or by the account key of the account that issued it.")
```

Comment:
It would be nice if we could avoid repeating this string, but I see why you're doing it.